src/codegen/revert.rs

```diff
@@ -57,11 +64,20 @@ impl SolidityError {
     }
 
     /// Return the selector of the error.
-    pub fn selector(&self) -> u32 {
+    pub fn selector(&self, ns: &Namespace) -> u32 {
```

Comment:
Using `u32` has endianness problems. Why not `[u8; 4]` or `Vec<u8>`

Reply:
> Using u32 has endianness problems

This u32 is only used internally in the compiler. Surely the Rust language does handle that correctly regardless for all plattforms right?

However, it's still a good point, it should be `[u8; 4]` and not `u32` as we never do arithmetics on selectors. It does also make it easier to work with in many places. Thanks!